### ISSUE TITLE
NameError: name 'patterns' is not defined

### ISSUE THREAD

**Issue by Sammeeey** [slyapustin/django-classified#59]
If I just add
```py
urlpatterns = patterns(
    url(r'', include('django_classified.urls', namespace='django_classified'),
)
```
to `uls.py` - as described in [Installation](https://github.com/slyapustin/django-classified#installation) - I get this error:
```
 )
     ^
SyntaxError: unexpected EOF while parsing
```

If I add an additional bracket at the end...
```py
urlpatterns = patterns(
    url(r'', include('django_classified.urls', namespace='django_classified'),
))
```
I get:
```
urlpatterns = patterns(
NameError: name 'patterns' is not defined
```

**Comment by damascene** [comment]
Not sure if this could help, I was able to view the main page without error with this, but unable to login:

```
from django.contrib import admin
from django.urls import path, re_path, include
# ~ from django.conf.urls import include, url

urlpatterns = [
    re_path('', include('django_classified.urls', namespace='django_classified')),
    path('admin/', admin.site.urls),
]
```

also had to add these apps in addition to the one in the documents:
```
    'django.contrib.sites',
    'django.contrib.humanize',
```

**Comment by slyapustin** [comment]
This was addressed in the #62
